etc/profile-a-l/imv.profile

```diff
@@ -0,0 +1,40 @@
+# Persistent local customizations
+include imv.local
+# Persistent global definitions
+include globals.local
+
+include allow-bin-sh.inc
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+
+include whitelist-runuser-common.inc
+
+apparmor
+caps.drop all
+net none
+nodvd
+noinput
```

Comment:
`nogroups`?

Reply:
After further testing, it turned out that what broke imv was actually file permission on images.

Some image files were not owned by the normal user. `nogroup` and `noroot` don't actually break imv.

Added it to my local copy.